Gemfile

```diff
@@ -51,9 +51,9 @@ group :development, :test do
 end
 
 group :issue_manager do
-  gem 'octokit',       '~> 1.25.0'
+  gem 'octokit',       '~> 3.8.0'
```

Comment:
old octokit uses old faraday api, so, we need to upgrade this to use the newer faraday api